demo/Makefile

```diff
@@ -0,0 +1,10 @@
+CC ?= gcc
+CXX ?= g++
+STRIP ?= strip
+
+all:
+	$(CC) login_cgi.c -o web_root/login.cgi -lfcgi -lcrypto
```

Comment:
Your sample login.cgi is not a FastCGI application.  The demo uses CGI.  The `-lfcgi` is not needed.

For demo purposes, the entire Makefile is not needed, either.  The README can contain the one line:
`cc login_cgi.c -o web_root/login.cgi -lcrypto`